vars/multicloudDeploy.groovy

```diff
@@ -114,17 +139,21 @@ void call(parameters = [:]) {
             error "Deployment skipped because no targets defined!"
         }
 
-        echo "Executing deployments"
-        if (config.parallelExecution) {
-            echo "Executing deployments in parallel"
-            parallel deployments
-        } else {
-            echo "Executing deployments in sequence"
-            def closuresToRun = deployments.values().asList()
-            Collections.shuffle(closuresToRun) // Shuffle the list so no one tries to rely on the order of execution
-            for (int i = 0; i < closuresToRun.size(); i++) {
-                (closuresToRun[i] as Closure)()
-            }
+        runClosures(config, deployments, "deployments")
+
+    }
+}
+
+def runClosures(Map config, Map toRun, String label = "closures") {
+    echo "Executing $label"
+    if (config.parallelExecution) {
+        echo "Executing $label in parallel"
+        parallel toRun
+    } else {
+        echo "Executing $label in sequence"
+        def closuresToRun = toRun.values().asList()
```

Comment:
In case shuffling is important we can/should shuffle here.

Reply:
No it is not important, and in the sdk pipeline we removed in meanwhile, thus also removing it here